ITISFoundation/osparc-simcore
# [bugfix] is1173/version bug

## What do these changes do?


Fixes the problem with the ``undefined`` commit short hash in the home page.
* Now we use ``--set-env`` to inject environmental variables into the frontend.
* Added little text in home page and app logo to indicate the current platform.

## Related issue number


#1173 

## How to test


* First make sure to use the latest ``qooxdoo-kit:latest`` image from dockerhub. Qx compiler must be updated.
* You can now see the commit hash in the home page

## Checklist
